src/Animations/Animations/Shared/Animations/Base/AnimationBase.cs

```diff
@@ -6,8 +6,13 @@ namespace Xamarin.Toolkit.Animations
     public abstract class AnimationBase : BindableObject
     {
         public static readonly BindableProperty TargetProperty =
-            BindableProperty.Create(nameof(Target), typeof(VisualElement), typeof(AnimationBase), null,
-                BindingMode.TwoWay, null);
+            BindableProperty.Create(
```

Comment:
not a fan of this, takes too much lines => less useful code on my screen

same remark can be made on quite a few files below

Reply:
The stylecop rule say : 
 - all parameters must be in one line
- or one parameter by line

Here => we have a mix 4 parameters in first ligne and 1 parameter in last line

Reply:
I think we have to think if we take the same rules of UWPCommunityToolkit or if we use custom rules for this project.

The UWPCommunityToolkit rules =>
https://github.com/Microsoft/UWPCommunityToolkit/blob/master/Toolkit.ruleset